doc/package.nix

```
@@ -32,16 +32,24 @@ let
     sed -i -e '/\*Declared by:\*/,+1d' $out
   '';
 in
-pkgs.stdenv.mkDerivation {
-  name = "wire-docs";
-  buildInputs = with pkgs; [
-    mdbook
-    mdbook-alerts
+pkgs.stdenv.mkDerivation (finalAttrs: {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Add `stdenv` to the top